lib/core/src/display/navigation/navigator.rs

```diff
@@ -80,11 +79,12 @@ impl Navigator {
         let camera_clone         = camera.clone();
         let mut properties_clone = properties.clone();
         let panning_callback     = move |pan: PanEvent| {
+            let y_scale = camera_clone.projection_matrix().m11;
```

Comment:
This is magic. It needs to be exposed as function from camera. With a good, good documentation.

Reply:
But the variable name is a documentation here, isn't it?

Reply:
Nope, the name doesn't tell me anything what it is about. What does `y_scale` mean for the camera?

Reply:
@wdanilo @farmaazon so... When we get perspective's matrix [1][1], we are getting y field of view slope. I have exposed this method in `Camera2d`:

```rust
    /// Gets y field of view slope.
    pub fn fovy_slope(&self) -> f32 {
        self.projection_matrix().m11
    }
```

Reply:
Ok, I can be dumb and just don't understand simple things (if so, I am sorry), but I still do not understand :( Few questions:
1. What is "y field of view slope"? I know what is field of view, but I dont know what is the "slope". Google isnt helpful here.
2. We obtain this "y slope" (whatever it is), why in the code belowe we do `let z = fov_slope;`? How the Y slope is related to the `z` component in the callback?

Reply:
I fixed the trigonometric formula.
```
    /|
   / | h
s /__|
   z
```
s = h / z

s = half fov y slope
z = the distance to cover all the screen we want to discover
h = half height

z = h / s

We can use `camera.half_fovy_slope()` which is equivalent to `(camera.fovy() / 2.0).tan()`, to get the slope value.